src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5291,6 +5291,7 @@ namespace HeatBalanceManager {
 
 				if ( MaxHeatLoadZone( ZoneNum ) > 1.0e-4 ) { // make sure load big enough to divide
 					MaxHeatLoadZone( ZoneNum ) = std::abs( max( MaxHeatLoadZone( ZoneNum ), MinLoad ) );
+					MaxHeatLoadPrevDay( ZoneNum ) = std::abs( max( MaxHeatLoadPrevDay( ZoneNum ), MinLoad ) );
```

Comment:
@EnergyArchmage would it help to initialize previous day load variables to 100 (instead of this?) ? Were you seeing this on the first pass through, or each time through?